obspy/taup/seismic_phase.py

```diff
@@ -146,37 +152,68 @@ def parse_name(self, tMod):
         currLeg = self.legs[0]
         nextLeg = currLeg
         isPWave = True
+        isPWavePrev = isPWave
```

Comment:
I think the first time `isPWavePrev` is different from `isPWave` is in [line 265](https://github.com/obspy/obspy/pull/1189/files#diff-fa794a8afc43a0792d57917b01f1557eR265) when it enters the loop. Before all occurrences of `isPWavePrev` can be removed I think (and replaced with `isPWave`). Or does it make sense for code readability and logic to keep it?